res/openvpn-gui-res-zh-hans.rc

```diff
@@ -98,7 +98,7 @@ LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
 BEGIN
     LTEXT "连接中…", ID_TXT_STATUS, 20, 5, 200, 10
     LTEXT "", ID_TXT_BYTECOUNT, 20, 175, 200, 10
-    LTEXT "OpenVPN GUI 11.y.0.0/2.4.x", ID_TXT_VERSION, 220, 175, 150, 10
+    LTEXT "OpenVPN GUI 11.y.0.0/2.5.6", ID_TXT_VERSION, 220, 175, 150, 10
```

Comment:
While this is okay, its not strictly necessary to edit this. Its overwritten by the correct version at runtime.

Reply:
ok